docs/source/plugins/developing_plugins.rst

```diff
@@ -2129,6 +2233,565 @@ plugin. Panels access these secrets via the `ctx.secrets` dict:
+
+                    Perhaps you would like to know some more information about your dataset?
+                """,
+                    name="markdown_screen_2",
+                )
+                table = types.TableView()
+
+                # set table columns
+                table.add_column("Dataset Name", label="Dataset Name")
+                table.add_column("Dataset Description", label="Description")
+                table.add_column("Number of Samples", label="Number of Samples")
+
+                panel.obj(
+                    name="info_table",
+                    view=table,
+                    label="Cool Info About Your Data",
+                )
+
+                add_panel_navigation(
+                    panel,
+                    left=True,
+                    right=True,
+                    on_left=self.go_to_previous_page,
+                    on_right=self.go_to_next_page,
+                )
+
+            elif ctx.panel.state.page == 3:
+
+                if ctx.panel.state.operator_status != "opened":
+                    stack.md(
+                        """
+                        ### One Last Trick
+
+                        If you want to do something cool, click the button below.
+                    """,
+                        name="markdown_screen_3",
+                    )
+                    btns = stack.obj("top_btns", view=button_container)
+                    btns.type.btn(
+                        "open_operator_io",
+                        label="Do Something Cool",
+                        on_click=self.open_operator_io,
+                    )
+
+                add_panel_navigation(
+                    panel,
+                    left=True,
+                    right=True,
+                    on_left=self.go_to_previous_page,
+                    on_right=self.go_to_next_page,
+                )
+            else:
+                stack.md(
+                    """
+                    #### How did you get here?
+                    Looks like you found the end of the walkthrough. Or have you gotten a little lost in the grid? No worries, let's get you back to the walkthrough!
+                """
+                )
+                btns = stack.obj("btns", view=button_container)
+                btns.type.btn("reset", label="Go Home", on_click=self.reset_page)
+
+            return types.Property(
+                panel,
+                view=types.GridView(
+                    height=100,
+                    width=100,
+                    align_x="center",
+                    align_y="center",
+                    componentsProps={
+                        "container": {"sx": {"position": "relative"}}
+                    },
+                ),
+            )
+
+
+    # Utility function to enhance styling of navigation buttons
+
+
+    def add_panel_navigation(
+        panel, left=True, right=False, on_left=None, on_right=None
+    ):
+        base_btn_styles = {
```

Comment:
@imanjra could you update this to show how to use the builtin next/prev buttons?